getting_started/beyond_the_basics.adoc

```diff
@@ -376,20 +376,16 @@ endif::[]
 == What's Next?
 
 The following sections provide some next steps now that you have finished your
-initial walkthrough of {product-title} (Next Gen) Developer Preview.
+initial walkthrough of {product-title} {product-version}.
 
 [[btb-developer-preview-considerations]]
 === Developer Preview Usage Considerations
 
-Please review the xref:../getting_started/devpreview_faq.adoc#getting-started-devpreview-faq[Developer Preview
```

Comment:
@sallyom Thanks for these changes! Yes, that is part of the review process -- identifying outdated and soon-to-be outdated references and removing them.